src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String color = ColorSupplier.getRandomColor();
+        int number = new Random().nextInt(100);
```

Comment:
Do not create a new Instance of `Random` every time, make it a class-level variable